notebook/static/tree/js/notebooklist.js

```diff
@@ -595,6 +597,20 @@ define([
             $('.delete-button').css('display', 'none');
         }
 
+        // View is visible when an item is renderable or downloadable
+        if (selected.length > 0 && !has_directory) {
```

Comment:
Ah, good point! HTML files seem like the one file type that should provide a "View" button. I'll try that now...